data/korath/korath outfits.txt

```diff
@@ -168,6 +168,21 @@ outfit "Systems Core (Large)"
 	"hull heat" 1.2
 	description `This massive bank of computers, shield generators, capacitors and repair microbot reservoirs holds everything needed to keep the very largest of Korath ships in working condition.`
 
+outfit "Korath Jump Drive"
+	category Systems
+	plural "Jump Drives"
+	cost 900000
+	thumbnail "outfit/jump drive korath"
+	"mass" 20
+	"outfit space" -20
+	"jump drive" 1
+	"jump speed" 0.3
+	"jump fuel" 200
+	"jump effect" "jump drive red"
+	"heat generation" 0.2
+	description "The workings of the jump drive are a total mystery. The best that human scientists have been able to find out is that this device is a containment field for matter in a state that cannot normally exist in this universe; if the drive is opened up, the field collapses and its contents disappear. Therefore, even though a few jump drives have been captured over the years, it has so far proved impossible to reverse engineer them."
+	description "Korath Jump Drives began as reverse-engineered Quarg Jump Drives. Therefore, they are less efficient, and produce a small amount of heat."
```

Comment:
This is not the case - the Korath Jump Drives are their own inventions, not reverse-engineered Quarg tech.

Reply:
Quatumshark beat me to it. But yes, the Korath are one of the very few races who have been able to independently invent Jump Drives for themselves. Some people have speculated that they might have been *inspired* by seeing Quarg use their Jump Drives, but even if that was the case, that merely taught them it is possible. There was no reverse engineering of Quarg drives of any kind.

Reply:
OK, I think I saw something about Korath Jump Drives being reverse-engineered Quarg but maybe I didn't pay enough attention. I'm working on a big commit that will fix this stuff, though.